dependencies.yaml

```diff
@@ -76,13 +77,33 @@ files:
       - docs
       - libarrow_run
       - py_version
+  # This is the shared library, bundled as a wheel. It is meant to be consumed by the wrapper.
+  py_rapids_build_libcudf:
+    output: pyproject
+    pyproject_dir: python/libcudf
+    extras:
+      table: build-system
```

Comment:
Great! We should also document this convention, and perhaps also write a backlog issue to lint for this kind of thing

Reply:
It would be good to document dependencies.yaml naming conventions in general somewhere, yes. Linting would be amazing but as you say probably long backlogged.

Reply:
I've applied this naming change here, and made it for all the other existing things in #16611.

Reply:
Works for me, thanks James.